resources/views/bread/edit-add.blade.php

```diff
@@ -56,6 +56,9 @@ class="form-edit-add"
                                     $options = json_decode($row->details);
                                     $display_options = isset($options->display) ? $options->display : NULL;
                                 @endphp
+                                @if (isset($options->legend->text))
```

Comment:
If `$options->legend` is not defined, I believe that it will complain about trying to get undefined property of null.

Reply:
I believe `isset()` should apply to the whole chain. https://stackoverflow.com/a/41099030